run-conduit-workload.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+CONDUIT_IMAGE=$1
+WORKLOAD=$2
+
+printf "\n-- Running %s with %s\n" "$WORKLOAD" "$CONDUIT_IMAGE"
+
+docker stop conduit-perf-test || true
+
+docker run --rm --name conduit-perf-test --memory 1g --cpus=2 -v "$(pwd)/plugins":/plugins -p 8080:8080 -d "$CONDUIT_IMAGE"
+sleep 1
+
+./run-workload.sh "http://localhost:8080" "$WORKLOAD"
```

Comment:
Is it worth having a separate file for this?

Reply:
Code-wise, I'd say not. But I've personally used it a few times as a shortcut (to execute the script, and then run the tool with the default options). Now that I think of it, I can default the Conduit URL to localhost:8080, to make it a bit shorter.